packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -770,10 +770,16 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 		const maybeUnusedStorageIds: Set<string> = new Set();
 		for (const route of blobRoutes) {
 			const blobId = getBlobIdFromGCNodePath(route);
+			// If the blob hasn't already been deleted, log an error because this should never happen.
+			// If the blob has already been deleted, log a telemetry event. This can happen because multiple GC
+			// sweep ops can contain the same data store. It would be interesting to track how often this happens.
+			const alreadyDeleted = this.isBlobDeleted(route);
 			if (!this.redirectTable.has(blobId)) {
-				this.mc.logger.sendErrorEvent({
+				this.mc.logger.sendTelemetryEvent({
```

Comment:
Why is this now just a Generic instead of an Error Event?

Reply:
The comment above explains. I added the "category" property explicitly. It's generic only if the blob has already been deleted. This can happen as there can be multiple sweep ops witht the same deleted node id.